tests/lxd/test_lxd_image_vault.cpp

```diff
@@ -410,25 +435,149 @@ TEST_F(LXDImageVault, has_record_for_returns_expected_values)
         return new mpt::MockLocalSocketReply(mpt::not_found_data, QNetworkReply::ContentNotFoundError);
     });
 
-    mp::LXDVMImageVault image_vault{hosts, mock_network_access_manager.get(), base_url};
+    mp::LXDVMImageVault image_vault{hosts, mock_network_access_manager.get(), base_url, mp::days{0}};
 
     EXPECT_TRUE(image_vault.has_record_for(instance_name));
     EXPECT_FALSE(image_vault.has_record_for("foo"));
 }
 
-TEST_F(LXDImageVault, unimplemented_functions_log_trace_message)
+TEST_F(LXDImageVault, update_image_refreshed_logs_expected_message)
```

Comment:
This verifies the refresh actually happens, too, no?

Reply:
It detects if `update_images()` requests a `refresh` which is what we are after, right?  After that, it would be up to LXD to do the actual refresh and since we are only mocking LXD here, we can do whatever.

Reply:
Sure, I just mean the test name :)

Reply:
Oh, right, I'll update the test name.

Reply:
Done.  Let me know if the name is better.